spec/models/vet360_redis/cache_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'common/exceptions'
+
+describe Vet360Redis::Cache do
+  let(:user) { build :user, :loa3 }
+  let(:contact_info) { Vet360Redis::ContactInformation.for_user(user) }
+
+  describe '.invalidate' do
+    it 'invalidates the vet360-contact-info-response cache' do
+      contact_info.cache(user.uuid, 'cache data')
+
+      expect_any_instance_of(Common::RedisStore).to receive(:destroy)
```

Comment:
@rlbaker @emersonthis - you'll note that I am testing that we are calling our Redis `destroy` method, in all of these cases.  I am not [re-testing that the `destroy` method works](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/lib/common/models/redis_store_spec.rb#L81-L108).

👍 ?